bugbash.md

```diff
@@ -79,7 +79,15 @@ All quickstarts and tutorials are in the [IoT Plug and Play documentation](https
 
 #### EmbeddedC and Azure RTOS
```

Comment:
How about "Embedded C SDK and Azure RTOS"

Reply:
sure